proxyd/consensus_poller.go

```diff
@@ -212,14 +215,17 @@ func (cp *ConsensusPoller) UpdateBackend(ctx context.Context, be *Backend) {
 	if !be.Online() || !be.IsHealthy() {
 		log.Warn("backend banned - not online or not healthy", "backend", be.Name)
 		cp.Ban(be)
+		return
```

Comment:
This is a logic change - is there an error that this fixes that wasn't covered by tests?